LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/Processors/LnsProtocolMessageProcessor.cs

```diff
@@ -72,10 +75,10 @@ internal class LnsProtocolMessageProcessor : ILnsProtocolMessageProcessor
 #pragma warning restore CA1508 // Avoid dead conditional code
                 {
                     // This can happen if the basic station client is losing connectivity
-                    this.logger.LogDebug(ex, ex.Message);
+                    this.logger.LogDebug(ex, "{Message}", ex.Message);
```

Comment:
This looks very strange where the whole message is a placeholder?

Reply:
What alternative would you suggest - suppress the warning?

Reply:
Reverted this with a29c611156a5b208419fac20e931f8f8661cce72. I suppressed the warning for now, since the merge of #842 generated many more of these instances. We will address this as part of a separate task (linked in the description). Since I already fixed some of these warnings in an older version of the branch, I will not revert those.